axum/src/extract/ws.rs

```diff
@@ -783,8 +783,9 @@ pub mod close_code {
     pub const PROTOCOL: u16 = 1002;
 
     /// Indicates that an endpoint is terminating the connection because it has received a type of
-    /// data it cannot accept (e.g., an endpoint that understands only text data MAY send this if
-    /// it receives a binary message).
+    /// data that it cannot accept.  
```

Comment:
Double spaces at the end of the line will add a line break (in addition to the new paragraph). I'm actually a bit surprised that rustfmt doesn't change these.

```suggestion
    /// data that it cannot accept.
```